d2graph/d2graph.go

```diff
@@ -1036,9 +1036,9 @@ func (obj *Object) GetDefaultSize(mtexts []*d2target.MText, ruler *textmeasure.R
 			}
 			maxTypeWidth = go2.Max(maxTypeWidth, typeDims.Width)
 
-			if c.Constraint != "" {
-				// covers UNQ constraint with padding
-				constraintWidth = 60
+			// 60 covers UNQ constraint with padding, 50 for further constraints covers UNQ + space
```

Comment:
can you add some e2e tests (just put it in stable_test) to see how this in action?